modules/nomad-clients/main.tf

```diff
@@ -13,7 +13,7 @@ data "aws_vpc" "selected" {
 # --------------------------------------------------------------------------------------------------
 
 module "nomad_clients" {
-  source = "github.com/hashicorp/terraform-aws-nomad//modules/nomad-cluster?ref=v0.5.0"
```

Comment:
I wonder if we should continue to keep track of the repo to take advantage of future releases 🤔  @guangie88 what do you think?

Reply:
Can just put the above URL back as a comment to remind ourselves I guess.